src/components/button/JustifyCenterButton.tsx

```diff
@@ -0,0 +1,26 @@
+// Chakra imports
+import { useColorModeValue, Button, Tooltip } from "@chakra-ui/react";
+import { MdFormatAlignJustify } from "react-icons/md";
+// Custom components
+
+export default function Default() {
+	const textColor = useColorModeValue("secondaryGray.900", "white");
+	const boxBg = useColorModeValue("secondaryGray.300", "whiteAlpha.100");
+
+	const handleJustifyCenterClick = () => {
+		document.execCommand("justifyCenter", true, "");
+	};
```

Comment:
Turns out this command is not supported on firefox, we already have a environment based error, lol

you can use the following code: (and find similar snippets for other commands)
```suggestion
	const handleAlignCenterClick = () => {
		if (document.queryCommandSupported("justifyCenter")) {
			document.execCommand("justifyCenter", false, null);
		} else {
			document.execCommand("styleWithCSS", false, "true");
			document.execCommand("justifyCenter", false, null);
			document.execCommand("styleWithCSS", false, "false");
		}
	};
```

Reply:
document.execCommand is deprecated (https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand#browser_compatibility) 
Should we reconsider using execCommand?